src/application/views/WelcomeView.tsx

```diff
@@ -1,52 +1,84 @@
 import { cx } from '../../lib/cx';
+import { Logo } from './Logo';
 
-export function WelcomeView({ hideShortcuts, className }: { hideShortcuts?: boolean; className?: string }) {
+export function WelcomeView() {
   return (
     <div
       className={cx(
-        'pointer-events-none select-none',
-        'flex w-full  flex-col items-center justify-center gap-2',
-        'bg-slate-300',
-        className,
+        'h-full w-full',
+        'flex flex-col items-center justify-center gap-28',
+        'px-6 pb-16 pt-0',
+        'bg-grayscale-10',
+        'cursor-default select-none',
       )}
     >
-      <div className="relative border border-slate-500 text-[160px] font-extrabold leading-none">
-        <WelcomeViewLetter className="bg-slate-500/50 text-slate-200/50" letter="R" />
-        <WelcomeViewLetter className="bg-slate-200/50 text-slate-500/50" letter="S" />
-      </div>
-
-      {!hideShortcuts && (
-        <div className="mt-10 space-y-4">
-          <WelcomeViewShortcut keys={['⌘', 'K']} text="Show Commands" />
-          <WelcomeViewShortcut keys={['⌘', 'P']} text="Show Files" />
-          <WelcomeViewShortcut keys={['⌘', 'R']} text="Show References" />
-          <WelcomeViewShortcut keys={['⌘', 'N']} text="New File" />
-          <WelcomeViewShortcut keys={['⌘', ',']} text="Show Settings" />
+      <Logo />
+      <div className="flex flex-row gap-20">
+        <div className="flex w-[16.5rem] flex-col gap-10">
```

Comment:
nit: `w-[16.5rem]`  - I don't like using [arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values) for widths but I understand how that is need for this design.